mm/kfence/core.c

```diff
@@ -525,7 +527,14 @@ bool kfence_handle_page_fault(unsigned long addr)
 		return kfence_unprotect(addr);
 	}
 
+	/*
+	 * If there is a KFENCE report somewhere inside lockdep, or one of the
+	 * libraries used by it, we need to avoid recursing back into lockdep.
+	 */
+	// TODO(elver): This is probably also a problem for allocations/frees.
+	lockdep_off();
```

Comment:
You are going to remove these annotations later, right?

Reply:
Yes I fixed it already (not pushed yet).